src/sql/src/parsers/tql_parser.rs

```diff
@@ -41,23 +94,31 @@ impl<'a> ParserContext<'a> {
         match self.parser.peek_token().token {
             Token::Word(w) => {
                 let uppercase = w.value.to_uppercase();
+                let _consume_tql_keyword_token = self.parser.next_token();
                 match w.keyword {
                     Keyword::NoKeyword
                         if (uppercase == EVAL || uppercase == EVALUATE)
                             && w.quote_style.is_none() =>
                     {
-                        let _ = self.parser.next_token();
                         self.parse_tql_eval().context(error::SyntaxSnafu)
                     }
 
                     Keyword::EXPLAIN => {
-                        let _ = self.parser.next_token();
-                        self.parse_tql_explain()
+                        let is_verbose = self.peek_token_as_string() == VERBOSE;
```

Comment:
Though `VERBOSE` is not supported in this PR -- I found that the parser only handles `TQL EXPLAIN VERBOSE` and won't match `TQL EXPLAIN verbose`.